splitio/models/splits.py

```diff
@@ -238,5 +249,6 @@ def from_raw(raw_split):
         raw_split.get('algo'),
         traffic_allocation=raw_split.get('trafficAllocation'),
         traffic_allocation_seed=raw_split.get('trafficAllocationSeed'),
-        configurations=raw_split.get('configurations')
+        configurations=raw_split.get('configurations'),
+        sets=raw_split['sets']
```

Comment:
why `raw_split['sets'] and not check `raw_split.get('sets')`

Reply:
will update it